frontend/src/pages/Home.tsx

```diff
@@ -0,0 +1,123 @@
+      <p>
+        Certified Interior Designers (CIDs) must fulfill CCIDC&#39;s education, experience, and exam
+        criteria to achieve the designation of &#34;Certified Interior Designer&#34;. CCIDC is
+        backed by The Certified Interior Designers Title Act which is governed by California
+        Business & Professions Codes Sections 5800-5812. CIDs, recognized as skilled professionals,
+        are authorized to develop and submit non-structural construction documents, demonstrating
+        expertise in the California Building Code. Upon qualification, CIDs receive a unique
+        identification number, a certificate, a stamp for nonstructural drawings, and an ID card,
+        symbolizing their unwavering commitment to upholding high professional standards and ethical
+        conduct.
+      </p>
+      <div className={styles.imgs}>
+        <div className={styles.imgCaption}>
+          <img alt="certificate" className={styles.cert} src={certificate} />
+          <p className={styles.caption}>CCIDC Certificate & Indentification Number</p>
+        </div>
+        <div className={styles.imgCaption}>
+          <img alt="stamp" src={stamp} />
+          <p className={styles.caption}>Individual Stamp</p>
+        </div>
+        <div className={styles.imgCaption}>
+          <img alt="id card" src={idcard} />
+          <p className={styles.caption}>Identification Card (ID)</p>
+        </div>
+      </div>
+      <p>
+        As the exclusive authority for approving the CID title, CCIDC establishes rigorous
+        eligibility criteria for achieving the title to ensure a high standard of qualification.
+        Becoming a CID is the exclusive and legally recognized title for interior designers in
+        California. CIDs demonstrate education, experience, and examination expertise tailored to
+        California&#39;s requirements, reducing legal risks and instilling heightened confidence in
+        clients. Accredited for professionalism, CIDs strictly adhere to the CCIDC Code of Ethics
+        and Conduct, emphasizing their unwavering commitment to maintaining the highest industry
+        standards.
+      </p>
+      <div className={styles.centeredContainer}>
+        <Button onClick={null}>Apply Now</Button>
+      </div>
+      <div className={styles.textDivider2}>
+        <div className={styles.heading}>Pathways to Certification</div>
+      </div>
+
+      <p>
+        To be eligible for the IDEX® California and begin the certification process, you must
+        provide evidence of completing the required education and/or diversified design work
+        experience in interior design or architectural work. You must meet one of the following path
+        criteria with proper qualifications listed below:
+      </p>
+
+      <div className={styles.pathwaysContainer}>
+        <Pathway
+          pathName="Path 1"
+          qualifications={[
+            "<40 semester core units from an accredited design program",
```

Comment:
I checked with Sujana - change to "< 40 semester core..."  (add a space between < and 40 for readability)